sql/moz-fx-data-shared-prod/telemetry/retention_clients/query.sql

```diff
@@ -0,0 +1,131 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.retention_clients`
```

Comment:
this one needs updating

Reply:
will update the view once we get the table fixed up.